CMakeLists.txt

```diff
@@ -99,14 +99,19 @@ find_package(VTK 9.2.6 REQUIRED
     RenderingVolumeOpenGL2
     TestingCore
     jsoncpp
-    opengl
   OPTIONAL_COMPONENTS
+    opengl
     IOExodus
     IOOpenVDB
     RenderingExternal
     RenderingRayTracing)
 message(STATUS "VTK ${VTK_VERSION} found")
 
+if(VTK_VERSION VERSION_LESS 9.3.20240914)
+  # Not optional before
+  find_package(VTK 9.0 REQUIRED COMPONENTS opengl)
```

Comment:
should be 9.2.6 too, or lets not specify the version at all

Reply:
nice catch